ceremonies/src/lib.rs

```diff
@@ -1431,12 +1434,15 @@ impl<T: Config> Pallet<T> {
 		for cid in cids {
 			if <IssuedRewards<T>>::iter_prefix_values((cid, cindex)).next().is_some() {
```

Comment:
I change this line in my debug branch because I think it does the same, but more concise:

```
if <IssuedRewards<T>>::contains_key((cid, cindex), 1) {
```

what do you think?

Reply:
no it does not the same. we want to check if there is ANY `meetup_index` in the map, not just index 1.

Reply:
But, aren't the meetup indexes starting at 1? Meaning the index 1, will always be populated if there is one meetup?

Reply:
well but it could be that the meetup at index 1 fails and all the others pass, right?

Reply:
Agreed, I am convinced 👍